core/controllers/collection_editor.py

```diff
@@ -150,8 +144,10 @@ def get(self, collection_id):
         except Exception as e:
             raise self.PageNotFoundException(e)
 
+        collection_dict['TAG_REGEX'] = feconf.TAG_REGEX
```

Comment:
OK for TAG_REGEX to be in GLOBALS. Let's handle that sort of thing separately -- we still need some way to pass constants to the frontend so that the constants in the frontend match those in feconf, and GLOBALS are the current way to do this -- though later we want to change this to use a JSON file instead.

On the other hand, all the rights stuff is not a constant, so they should go in the ajax handlers.